token-group/js/src/state/tokenGroup.ts

```diff
@@ -0,0 +1,66 @@
+import { PublicKey } from '@solana/web3.js';
+import { getBytesCodec, getStructCodec } from '@solana/codecs-data-structures';
+
+const tokenGroupCodec = getStructCodec([
+    ['updateAuthority', getBytesCodec({ size: 32 })],
+    ['mint', getBytesCodec({ size: 32 })],
+    ['size', getBytesCodec({ size: 4 })],
+    ['maxSize', getBytesCodec({ size: 4 })],
+]);
```

Comment:
We can re-work this codec a bit to no longer require that `u32` conversion function, and it will just happen automatically with `encode` or `decode`.

`getU32Codec` can be found in `@solana/codecs-numbers`.

```suggestion
const tokenGroupCodec = getStructCodec([
    ['updateAuthority', getBytesCodec({ size: 32 })],
    ['mint', getBytesCodec({ size: 32 })],
    ['size', getU32Codec()],
    ['maxSize', getU32Codec()],
]);
```

Reply:
awesome! updated